docs/fspiop-rest-v1.0-OpenAPI-implementation_openapi3.yaml

```diff
@@ -0,0 +1,3965 @@
+        amount:
+          $ref: '#/components/schemas/Money'
+          description: Transaction amount to be sent.
+        transactionType:
+          $ref: '#/components/schemas/TransactionType'
+          description: Type of the transaction.
+        note:
+          type: string
+          description: Memo associated to the transaction, intended to the Payee.
+        extensionList:
+          $ref: '#/components/schemas/ExtensionList'
+          description: Optional extension, specific to deployment.
+      required:
+      - transactionId
+      - quoteId
+      - payee
+      - payer
+      - amount
+      - transactionType
+    TransactionRequestsIDPutResponse:
+      title: TransactionRequestsIDPutResponse
+      type: object
+      description: The object sent in the PUT /transactionRequests/{ID} callback.
+      properties:
+        transactionId:
+          type: string
+          description: Identifies a related transaction (if a transaction has been created).
+          example: b51ec534-ee48-4575-b6a9-ead2955b8069
+        transactionRequestState:
+          type: string
+          description: State of the transaction request.
+          example: RECEIVED
+        extensionList:
+          $ref: '#/components/schemas/ExtensionList'
+          description: Optional extension, specific to deployment.
+      required:
+      - transactionRequestState
+    TransactionRequestsPostRequest:
+      title: TransactionRequestsPostRequest
+      type: object
+      description: The object sent in the POST /transactionRequests request.
+      properties:
+        transactionRequestId:
+          type: string
+          description: Common ID between the FSPs for the transaction request object, decided by the Payee FSP. The ID should be reused for resends of the same transaction request. A new ID should be generated for each new transaction request.
+          example: b51ec534-ee48-4575-b6a9-ead2955b8069
+        payee:
+          $ref: '#/components/schemas/Party'
+          description: Information about the Payee in the proposed financial transaction.
+        payer:
+          $ref: '#/components/schemas/PartyIdInfo'
+          description: Information about the Payer type, id, sub-type/id, FSP Id in the proposed financial transaction.
+        amount:
+          $ref: '#/components/schemas/Money'
+          description: Requested amount to be transferred from the Payer to Payee.
+        transactionType:
+          $ref: '#/components/schemas/TransactionType'
+          description: Type of transaction.
+        note:
+          type: string
+          description: Reason for the transaction request, intended to the Payer.
+          example: Free-text memo.
+        geoCode:
+          $ref: '#/components/schemas/GeoCode'
+          description: Longitude and Latitude of the initiating Party. Can be used to detect fraud.
+        authenticationType:
+          type: string
+          description: OTP or QR Code, otherwise empty.

```

Comment:
Update this description for U2F

Reply:
I have updated that description in athorization related methods but not in TransactionRequests .
Will do now. Thanks

Reply:
Ok updated authenticationType description for U2F.